tracer/test/snapshots/Security.AspNetCore2.SecurityDisabled.TestUserLoginEvent_eventName=loginevent.auto.failure_bodyString=Input.Email=no_such_user@test.com&Input.Password=test.verified.txt

```diff
@@ -0,0 +1,152 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet_core.request,
+    Resource: POST /account/index,
```

Comment:
I seem to be missing something here, shouldn't these have the login tags added 🤔

Reply:
this snapshot is when Security is disabled so user events shouldnt be collected  (tracer/test/snapshots/Security.AspNetCore2.**SecurityDisabled**.TestUserLoginEvent_eventName=loginevent.auto.failure_bodyString=Input.Email=no_such_user@test.com&Input.Password=test.verified.txt)

Reply:
🤦‍♂️  my bad, I couldn't see that in the URL for the life of me 😂